source/includes/_resource.collections.md

```diff
@@ -69,13 +69,15 @@ vhx.collections.create();
 $ curl -X POST "https://api.vhx.tv/collections" \
   -d name="Collection Name" \
   -d type="series" \
+  -d site="http://api.vhx.tv/sites/:id"
```

Comment:
i'm trying to rid all references to `site`. can you use `product` as an example context here?